portals/knowledge-management/src/Components/Article.vue

```diff
@@ -54,14 +54,13 @@
         }"
         class="group p-3 grid space-y-2 text-sm font-medium text-gray-700"
     >
-        <Tags :tags="article.tags" />
-
         <div class="flex">
             <span> {{ article.name }} </span>
             <ChevronRightIcon
                 class="opacity-0 h-5 w-5 text-primary-600 transition-all group-hover:translate-x-2 group-hover:opacity-100"
             />
         </div>
+        <Tags :tags="article.tags" class="text-xs font-medium me-2 py-0.5 rounded" />
```

Comment:
Is there a particular reason we chose to put the class changes here and not in the `Tags` component itself?

Reply:
Hi, 
Fixed.
Thank You!!